src/main/resources/react/src/pages/UserInfo/tab_pages/my_profile/myProfile.js

```diff
@@ -0,0 +1,195 @@
+import React, { useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import TitleBigBlue from '../../../../components/title_Big_Blue';
+import InputProfile from '../../components/inputProfile/inputProfile';
+import BlockButtons from '../../components/buttonsAddRemoveChild/blockButtons';
+import Button from '../../../../components/button/Button';
+import InputGender from '../../components/inputProfile/inputGender';
+import CheckBox from '../../../../components/checkbox';
+
+import './myProfile.scss';
+
+import {
+	addChild,
+	addChildrenInputValue,
+	addMeInputValue, addPhone,
+	addPhoneInputValue,
+	deleteChild, deletePhone,
+	putUserInfoAsync
+} from '../../../../redux/profile/profileAction';
+
+
+const MyProfile = () => {
+	const dispatch = useDispatch();
+	const { profile } = useSelector( state => state.profileMe );
+
+	const addInputMe = useCallback( ( e ) => {
+		dispatch( addMeInputValue( { [e.target.name]: e.target.value } ) );
+	}, [ dispatch ] );
+
+	const addInputCity = ( e ) => {
+		console.log( { [e.target.name]: e.target.value } );
```

Comment:
We should think about logging all front errors on the server as well. Please remind me about it on the next team meeting.